### PR TITLE
Add thepiratebay.org

### PR DESCRIPTION
**Pull request by Leichesters**
imgs should be inverted

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -203,7 +203,8 @@ path[d^="M34.8"]
 path[d^="M15.759"]
 
 CSS
-a, abbr, acronym, address, applet, article, aside, audio, b, big, blockquote, body, canvas, caption, center, cite, code, dd, del, details, dfn, div, dl, dt, em, embed, fieldset, figcaption, figure, footer, form, h1, h2, h3, h4, h5, h6, header, hgroup, html, i, iframe, img, ins, kbd, label, legend, li, main, mark, menu, nav, object, ol, output, p, pre, q, ruby, s, samp, section, small, span, strike, strong, sub, summary, sup, table, tbody, td, tfoot, th, thead, time, tr, tt, u, ul, var, video {
+a, abbr, acronym, address, applet, article, aside, audio, b, big, blockquote, body, canvas, caption, center, cite, code, dd, del, details, dfn, div, dl, dt, em, embed, fieldset, figcaption, figure, footer, form, h1, h2, h3, h4, h5, h6, header, hgroup, html, i, iframe, img, ins, kbd, label, legend, li, main, mark, menu, nav, object, ol, output, p, pre, q, ruby, s, samp, section, small, span, strike, strong, sub, summary, sup, table, tbody, td, tfoot, th,
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:206]: Please revert this change.

Comment (Leichesters) [reply src/config/dynamic-theme-fixes.config]: sorry about that, should be fixed now

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -20087,6 +20088,13 @@ INVERT
 
 ================================
 
+thepiratebay.org
```
[the remark this answers is not in the document]
Comment (Leichesters) [reply src/config/dynamic-theme-fixes.config:20090]: Hey Gusted, I guess you're pointing that thepiratebay would be illegal. thepiratebay itself is not illegal, torrenting copyrighted content is illegal, but there is also legal stuff like linux isos or copyright expired content on thepiratebay.

Okay now back to your concern regarding inverting all images.

thepiratebay basically only contains the logo as an image.

Here are some screenshots for proof:

image

image

image

[the remark this answers is not in the document]
Comment (Leichesters) [reply src/config/dynamic-theme-fixes.config:20090]: although inverting seems a bit too much. how can we partially invert it so it matches the background?

[the remark this answers is not in the document]
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:20090]: > Hey Gusted, I guess you're pointing that thepiratebay would be illegal. thepiratebay itself is not illegal, torrenting copyrighted content is illegal, but there is also legal stuff like linux isos or copyright expired content on thepiratebay.

That's not the problem, the problem is [the ISPs](https://torrentfreak.com/dutch-isps-must-block-pirate-bay-proxies-and-mirrors-again-court-rules-201015/). Anyhow that aside, I can trust my gust to see if the CSS is reasonable 😅.

> thepiratebay basically only contains the logo as an image.

Is it possible to just target the logo then?

[the remark this answers is not in the document]
Comment (Leichesters) [reply src/config/dynamic-theme-fixes.config:20090]: > That's not the problem, the problem is [the ISPs](https://torrentfreak.com/dutch-isps-must-block-pirate-bay-proxies-and-mirrors-again-court-rules-201015/). Anyhow that aside, I can trust my gust to see if the CSS is reasonable sweat_smile.

You can try changing your dns settings to 1.1.1.1, 8.8.8.8 or 8.8.4.4 or use a vpn

> Is it possible to just target the logo then?

Yes should be fixed now

[the remark this answers is not in the document]
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:20090]: > You can try changing your dns settings to 1.1.1.1, 8.8.8.8 or 8.8.4.4 or use a vpn

Yeah I'm aware, but I'd prefer avoiding dangerous waters as the ISP ultimately still see the IP I'm requesting with changing DNS servers.

**Review by Gusted — APPROVED** [review]
